### PR TITLE
436: draft documents link

### PR DESCRIPTION
**Pull request by sutt0n**
* Note: added `CREATE_ORDER_DOCKET_ENTRY` permission to Docket Clerk role per design

### CODE REVIEW

File: web-client/src/presenter/computeds/formattedCaseDetail.js [hunk 1]
```diff
@@ -178,10 +178,16 @@ export const formattedCaseDetail = (get, applicationContext) => {
 
   result.formattedDraftDocuments = (result.draftDocuments || []).map(
     draftDocument => {
+      let editLink = '';
+
+      if (permissions.DOCKET_ENTRY && permissions.CREATE_ORDER_DOCKET_ENTRY) {
```
Comment (matthopson) [inline web-client/src/presenter/computeds/formattedCaseDetail.js]: Why the new permission? Why not just DOCKET_ENTRY? Or is Clerk of the Court not meant to be able to do this?

Comment (sutt0n) [reply web-client/src/presenter/computeds/formattedCaseDetail.js]: Just the docket clerk and petitions clerk. The corresponding interactor for the `add-court-issued-docket-entry` checks for both `DOCKET_ENTRY` and `CREATE_ORDER_DOCKET_ENTRY` above.

Comment (matthopson) [reply web-client/src/presenter/computeds/formattedCaseDetail.js]: Petitions clerk doesn't have this permission.

Comment (matthopson) [reply web-client/src/presenter/computeds/formattedCaseDetail.js]: Oh, you meant the other permission.

**Review by matthopson — APPROVED** [review]

**Pull request merged**